packages/webpack-cli/lib/groups/basicResolver.js

```diff
@@ -0,0 +1,52 @@
+const logger = require('../utils/logger');
+const { core, groups } = require('../utils/cli-flags');
+const { resolveFilePath } = require('../utils/arg-utils');
+
+const WEBPACK_OPTION_FLAGS = core
+    .filter((coreFlag) => {
+        return coreFlag.group === groups.BASIC_GROUP;
+    })
+    .reduce((result, flagObject) => {
+        result.push(flagObject.name);
+        if (flagObject.alias) {
+            result.push(flagObject.alias);
+        }
+        return result;
+    }, []);
+
+console.log(WEBPACK_OPTION_FLAGS);
+
+function resolveArgs(args) {
+    const finalOptions = {
+        options: {},
+        outputOptions: {},
+    };
+    Object.keys(args).forEach((arg) => {
+        if (WEBPACK_OPTION_FLAGS.includes(arg)) {
+            finalOptions.outputOptions[arg] = args[arg];
+        }
+        // TODO: to add once webpack bundle analyzer is installed, which is not at the moment
+        // if (arg == 'analyze') {
+        // const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+        // this.opts.options.plugins = [new BundleAnalyzerPlugin()];
+        // }
```

Comment:
Let's remove this, we need to open new issue and do some work on `webpack-bundle-analyzer` side

Reply:
yep, did some work https://github.com/webpack/webpack-cli/pull/1361 will revisit later

Reply:
What's the work needed on `webpack-bundle-analyzer` side @evilebottnawi ?

Maybe I'll open an issue about it

Reply:
Let's do it late, just open a new issue